### PR TITLE
Polish translation pt. 1

### PR DESCRIPTION
**Pull request by ByteBolt556**
This is my first time doing a pull request so please forgive me if I messed something up. I am willing to fix anything if asked 😁

### CODE REVIEW

File: contracts/pl.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "generic_goodluck": "Powodzenia!",
+  "generic_objectives": "Aby zawrzeć tę umowę, musisz:",
```
Comment (Elanis) [inline contracts/pl.json]: Can you confirm this is the right verb to mean "fulfill" or "complete" ?

File: contracts/pl.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "generic_requirements": "Ten kontrakt pojawi się w grze, jeśli spełnisz następujące wymagania:",
+  "generic_reward": "Za ukończenie tego kontraktu otrzymasz:",
```
Comment (Elanis) [inline contracts/pl.json:5]: The french/english translations give me a weird sentence, so to be sure. Does it mean that's what you get when you end the contract ?

File: contracts/pl.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "first_production-subtitle": "Zbuduj tartak i wytwarzaj deski.",
+  "first_production-description": "Na początek możesz spróbować pociąć te drewniane kłody na deski i je sprzedać!",
+  "manufacturinc_follow_objective-title": "Obserwuj Manufactur'inc na Steamie",
+  "manufacturinc_follow_objective-subtitle": "Obserwuj stronę gry na Steamie i zdobądź nagrodę!",
+  "manufacturinc_follow_objective-description": "Obserwuj stronę gry na Steamie i zdobądź nagrodę!",
+  "recycle_woodchip-title": "Poddaj wióry drzewne recyklingowi",
+  "recycle_woodchip-subtitle": "Przy ścinaniu drzew powstają wióry. Wykorzystajmy je!",
+  "recycle_woodchip-description": "Podczas ścinania drzew powstają wióry. Wykorzystajmy je do stworzenia czegoś nowego!",
+  "everyone_needs_boxes-title": "Każdy potrzebuje pudełek",
+  "everyone_needs_boxes-subtitle": "Każdy potrzebuje pudełek do każdego celu, więc wyprodukujmy ich trochę!",
```
Comment (Elanis) [inline contracts/pl.json]: I think we can simplify those two sentences. Can you write "Everyone needs boxes ~~for every purpose~~, so let's produce some!" ?

File: contracts/pl.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "everyone_needs_boxes-description": "Każdy potrzebuje pudełek do każdego celu, więc wyprodukujmy ich trochę!",
+  "higher_planks_output-title": "Większa produkcja desek",
+  "higher_planks_output-subtitle": "Zwiększmy wydajność produkcji desek.",
+  "higher_planks_output-description": "Zwiększmy wydajność produkcji desek.",
+  "burn_baby_burn-title": "♪ Burn Baby Burn ♫",
+  "burn_baby_burn-subtitle": "Spalanie drewna tworzy węgiel, a my potrzebujemy!",
```
Comment (Elanis) [inline contracts/pl.json]: To be sure, is this charcoal and not coal/carbon ? (Sometimes languages have different words 😅 )

File: store/pl.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  "short-description": "Poprowadź swoją nową firmę do sukcesu w tej grze o zarządzaniu branżą. Zaplanuj i zoptymalizuj swoją produkcję; podejmuj etyczne decyzje, aby wpłynąć na warunki pracy, wpływ na środowisko, poziom automatyzacji i wiele więcej. Czy stworzysz prostą produkcję czy PROdukcję?",
```
Comment (Elanis) [inline store/pl.json:3]: I love the pun here!

**Review by Elanis — COMMENTED** [review]
Wow, just, thank you for this work!

I put some comments with questions :) Since I don't speak polish, I used various translation tools to translate from your language to french and english so I was able to check your strings.

Thanks for this!

NB: I forgot to enable the CLA on this repo, I will do it asap (it's needed for me to be legally protected on the game. Not the funniest part 😅 )

File: contracts/pl.json (hunk 1 above)
Comment (ByteBolt556) [reply contracts/pl.json]: Okay when it comes to Polish - the context is important. I basically wrote "to sign" or "to accept" this contract. This has been changed to "complete" the contract

File: contracts/pl.json (hunk 2 above)
Comment (ByteBolt556) [reply contracts/pl.json:5]: Yes. It means "For resolving this contract you will recieve:"

File: contracts/pl.json (hunk 3 above)
Comment (ByteBolt556) [reply contracts/pl.json]: Done!

File: contracts/pl.json (hunk 4 above)
Comment (ByteBolt556) [reply contracts/pl.json]: In polish carbon and charcoal have the same word for it which is indeed **"węgiel"**

File: store/pl.json (hunk 5 above)
Comment (ByteBolt556) [reply store/pl.json:3]: I know right? I understand the pun you wrote in English but it didn't work in Polish so I had to come up with something 🤣

File: contracts/pl.json (hunk 4 above)
Comment (Elanis) [reply contracts/pl.json]: Great, thanks :)

File: store/pl.json (hunk 5 above)
Comment (Elanis) [reply store/pl.json:3]: Yep, it's well done !

**Comment by Elanis** [comment]
Everything seems fine for me :) Thanks a lot for your contribution, you can't imagine how I appreciate ❤️ 

One last question before merging, I like to credit people who translate my games.
Do you want to be in the Credits ? And if yes, which name should I use in the credits? "ByteBolt556" ?

**Comment by ByteBolt556** [comment]
> Everything seems fine for me :) Thanks a lot for your contribution, you can't imagine how I appreciate ❤️
> 
> One last question before merging, I like to credit people who translate my games. Do you want to be in the Credits ? And if yes, which name should I use in the credits? "ByteBolt556" ?

Yes of course I'd love to be in credits. You can write "Laggex (ByteBolt556)" in credits. I'd appreciate it 😄

**Comment by Elanis** [comment]
Great! This will probably be embed in the next (0.3.5) update then :)

Thanks again!

**Pull request merged**